.vscode/settings.json

```diff
@@ -1,6 +1,3 @@
 {
-    "files.exclude": {
```

Comment:
Not sure what you mean here, but I was constantly checking stuff in `bazel-out` for sanity-checking, but I'm also not ready to land this yet.

In general, both of these seem like settings that would make more sense in a personal setting and not on a repo level.

Reply:
> In general, both of these seem like settings that would make more sense in a personal setting and not on a repo level.

Yeah well put - these both seem like GUI preferences

Reply:
I'm not opposed to removing it. I think it was there because when you search for stuff you keep getting hits in the `bazel-out` directory but like @dierksen said, it can be a personal preference.